commands/previewPush.go

```diff
@@ -129,13 +153,30 @@ func (args *PushArgs) flags() []cli.Flag {
 	return flags
 }
 
+const pwarn = "WARNING: This command will receive a major upgrade in the next release (v4.15). Try the new behavior by using ppreview/ppush. Please test and report any bugs ASAP. The older code will be available as oldpreview/oldpush for a limited number of releases."
+const owarn = "WARNING: oldpreview/preview are temporary commands and may go away in a future release without warning. They are not subject to SemVer. Please upgrade to ppreview/ppush instead."
```

Comment:
Naming things and documentation is difficult, so be extra careful witxh choosing the right terms. Do you mean by 'Please upgrade to ppreview/ppush instead' that everyone should change their processes/code to `ppreview` and `ppush`? I would expect a swap from `preview` (serial) to `preview` (concurrent) in upcoming release(s) where our processes/code remains unchanged. The other hint about testing with `ppreview`/`ppush` certainly seems useful to me.